Remotion/Web/Web.Development.WebTesting.IntegrationTests.RequireUI/BrowserWindowTest.cs

```diff
@@ -0,0 +1,75 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+//
+// The re-motion Core Framework is free software; you can redistribute it
+// and/or modify it under the terms of the GNU Lesser General Public License
+// as published by the Free Software Foundation; either version 2.1 of the
+// License, or (at your option) any later version.
+//
+// re-motion is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+//
+using System;
+using System.Drawing;
+using NUnit.Framework;
+using Remotion.Web.Development.WebTesting.PageObjects;
+using Remotion.Web.Development.WebTesting.Utilities;
+
+namespace Remotion.Web.Development.WebTesting.IntegrationTests.RequireUI
+{
+  [TestFixture]
+  public class BrowserWindowTests : IntegrationTest
+  {
+    protected override bool MaximizeMainBrowserSession
+    {
+      get { return false; }
+    }
+
+    public BrowserHelper BrowserHelper
+    {
+      get { return Helper.BrowserConfiguration.BrowserHelper; }
+    }
+
+    [Test]
+    public void Test_ContentPosition ()
```

Comment:
I think we should copy the other two tests, too sicne it makes a difference whether we're using UI or headless when moving the window / resizing the window.